homeassistant/components/tuya/strings.select.json

```
@@ -80,6 +80,11 @@
         "point": "Point",
         "part": "Part",
         "pick_zone": "Pick Zone"
+    },
+    "tuya__fan_angle": {
+        "30": "30° angle",
+        "60": "60° angle",
+        "90": "90° angle"
```

"angle" is already part of the entity name?

Maybe just:

```suggestion
        "30": "30°",
        "60": "60°",
        "90": "90°"
```